app/controllers/items_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ItemsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @item = Item.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @items = @user.items
+    @item = current_user.items.build(item_params)
+    @item.user = @user
+    @new_item = Item.new
```

Comment:
I used this in the create.js.erb (followed bloccit example - it works) probably why I had in there - 

```
<% if @item.valid? %>
  $('.js-items').prepend("<%= escape_javascript(render(@item)) %>");
  $('.new-item').html("<%= escape_javascript(render partial: 'items/form', locals: { user: @user, item: @new_item }) %>");
<% else %>
  $('.new-item').html("<%= escape_javascript(render partial: 'items/form', locals: { user: @user, item: @item }) %>");
<% end %>
```